### DISCUSSION TITLE
[Esame 22 gennaio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#444] [Esami M-Z]
![22 gennaio es. 3](https://github.com/user-attachments/assets/acc445c0-ecfd-4c42-baeb-8bf2c93747f2)

<details>
  <summary>soluzione del prof</summary>
  <br>
 <img src="https://github.com/user-attachments/assets/81c128fe-3126-41e5-b362-9a7090ffb9b3" alt="soluzione"  />
  
</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/52fcf972-e36e-4d72-b592-9a965412e45f)

**Comment by niccolo-fato** [comment]
<img width="635" height="1280" alt="image" src="https://github.com/user-attachments/assets/55ba32c1-ea82-4720-a775-c6be0104ef8c" />
